src/status_im/contexts/wallet/events.cljs

```diff
@@ -84,6 +84,30 @@
                                    {:error %
                                     :event :wallet/save-account})}]]]}))
 
+(rf/reg-event-fx
+ :wallet/show-account-deleted-toast
+ (fn [_ [message]]
+   {:dispatch [:toasts/upsert {:type :positive :text message}]}))
```

Comment:
nest to align maps keys 👍

Reply:
also the suggested approach is using `:fx` with :dispatch inside